README.md

```diff
@@ -72,9 +72,9 @@ Face detection models identify and/or recognize human faces in images. Some more
 
 |Model Class |Reference |Description |
 |-|-|-|
-|<b>[ArcFace](models/face_recognition/ArcFace/)</b>|[Deng et al.](https://arxiv.org/abs/1801.07698)|ArcFace is a CNN based model for face recognition which learns discriminative features of faces and produces embeddings for input face images.|
+|<b>[ArcFace](Image/Body_Face_and_Gesture_Analysis/ArcFace)</b>|[Deng et al.](https://arxiv.org/abs/1801.07698)|ArcFace is a CNN based model for face recognition which learns discriminative features of faces and produces embeddings for input face images.|
```

Comment:
consider shortening to "body_analysis" instead of "Body_Face_and_Gesture_Analysis"